### ISSUE TITLE
A11y_Visual Studio Code Web Extensions_Github_Description_Keyboard: On closing the "Do you want Code - Insiders to open the external website?" popup, the focus does not return to the 'Image' present under the text before associated with the comment shared by Thomus1664

### ISSUE THREAD

**Issue by sspatil123** [microsoft/vscode-pull-request-github#4242]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:**#A11yMAS;#A11yTCS;#GH_Visual Studio Code Web Extensions_Nov2022;#Visual Studio Code Web Extensions ;#ChromiumEdge;#DesktopWeb;#A11ySev3;#Win11;#WCAG2.4.3;#Keyboard;#External;#External:GitHub;#External:Bug170442;

## Environment Details:
Application Name: Visual Studio Code Web Extensions
URL: https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github
Windows Version: Win11
Edge version:107.0.1418.56

## Repro Steps:

1. Please do not close this bug. This bug should only be closed by TCS, C+AI Accessibility tester after verification of the fix
2. Open https://insiders.vscode.dev/github/microsoft/vscode-pull-request-github environment and login with git hub account.
3. tab to 'Github' and press enter.
4. tab to 'All Open' and press enter.
5. Tab to any pull request and press enter 'Description'.
6. Tab till 'Image' link and press enter.
7. Check after closing dialog focus is return back to same control or not.

## Actual:
On closing the "Do you want Code - Insiders to open the external website?" popup, the focus does not return to the 'Image' present under the text before associated with the comment shared by Thomus1664.

## Expected:
On closing the "Do you want Code - Insiders to open the external website?" popup, the focus Should be return to the 'Image' present under the text before associated with the comment shared by Thomus1664.

## User Impact:
Keyboard user will not get the right order of navigation back to the 'Image' link.

## Attachment:
![FocusOrderIncorrect](https://user-images.githubusercontent.com/85863999/203890393-37a0984a-41b8-489f-a0a6-adb7f3855439.gif)

**Comment by fsteffi** [comment]
external bug:https://github.com/microsoft/vscode/issues/170442

**Comment by alexr00** [comment]
Closing since this is not an issue in the extension and instead an issue with VS Code.
